src/Compilers/CSharp/CscCore/csc.deps.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
This is the file that the shared framework host uses to resolve dependencies from the shared framework.